### PR TITLE
Cool favorite toggle animation ❤️ 

### LINKED ISSUE #283: Cool favorite toggle animation
## Kind (Required)

- Improvement

 ## Overview (Required)

- Do you have any favorite animations?

 ## Links	

- https://developer.android.com/jetpack/compose/animation

### PR DESCRIPTION
**Pull request by fornewid**
## Issue
- close #283

- Create `FavoriteAnimation` composable and add animation to each feed item.
- To improve the visual effect, change 'favorite' color to **red**.

## Links
- https://developer.android.com/jetpack/compose/animation
- References
  - [HeartAnimationView.kt](https://github.com/fornewid/android-animation-11p-more/blob/end/sample/src/main/java/soup/animation/sample/widget/HeartAnimationView.kt)
  - [LottieAnimation.kt](https://github.com/airbnb/lottie-android/blob/master/lottie-compose/src/main/java/com/airbnb/lottie/compose/LottieAnimation.kt)

## Screenshot

https://user-images.githubusercontent.com/3405740/110671743-3844fe80-8212-11eb-9494-e27c8af133a7.mp4

### CODE REVIEW

**Comment by takahirom** [comment]
👀

File: uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FavoriteAnimation.kt [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        val width = size.width
+        val height = size.height
+        hearts.fastForEach { heart ->
+            val transX = lerp(drawableHalfWidth, width - drawableHalfWidth, heart.transX)
+            val transY = lerp(drawableHalfHeight, height - drawableHalfHeight, heart.transY)
+            translate(transX, transY) {
+                scale(scaleX = heart.scale, scaleY = heart.scale, pivot = Offset(0.5f, 0.5f)) {
+                    drawIntoCanvas { canvas ->
+                        drawable.alpha = (255 * heart.alpha).toInt()
+                        drawable.draw(canvas.nativeCanvas)
```
Comment (takahirom) [inline uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FavoriteAnimation.kt:99]: I didn't know that we can use `canvas.nativeCanvas`!!

**Review by takahirom — APPROVED** [review]
Thank you! LGTM 👍

File: uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FavoriteAnimation.kt (hunk 1 above)
Comment (fornewid) [reply uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FavoriteAnimation.kt:99]: Yes, we can use!

**Pull request merged**